frontend/lib/src/components/elements/ArrowTable/styled-components.ts

```diff
@@ -22,17 +22,8 @@ export const StyledTableContainer = styled.div(({ theme }) => ({
   fontSize: theme.fontSizes.md,
   fontFamily: theme.genericFonts.bodyFont,
   lineHeight: theme.lineHeights.small,
-  overflow: ["auto", "overlay"],
-}))
 
-export const StyledTable = styled.table(({ theme }) => ({
-  width: theme.sizes.full,
-  marginBottom: theme.spacing.lg,
```

Comment:
Removed this because all other elements also don't have top/bottom margin. Not sure if there's a specific reason for having it here.